packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2446,7 +2446,6 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   @pragma('vm:notify-debugger-on-exception')
   void _layoutWithoutResize() {
     assert(_needsLayout);
-    assert(_relayoutBoundary == this);
```

Comment:
Instead of remove this, can it add a `|| this is RenderObjectWithLayoutCallbackMixin`